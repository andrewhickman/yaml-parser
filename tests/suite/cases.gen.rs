case!(case_229q, r#".\229Q"#);
case!(case_236b, r#".\236B"#, fail: true);
case!(case_26dv, r#".\26DV"#);
case!(case_27na, r#".\27NA"#);
case!(case_2auy, r#".\2AUY"#);
case!(case_2cms, r#".\2CMS"#, fail: true);
case!(case_2ebw, r#".\2EBW"#);
case!(case_2g84_00, r#".\2G84\00"#, fail: true);
case!(case_2g84_01, r#".\2G84\01"#, fail: true);
case!(case_2g84_02, r#".\2G84\02"#);
case!(case_2g84_03, r#".\2G84\03"#);
case!(case_2jqs, r#".\2JQS"#);
case!(case_2lfx, r#".\2LFX"#);
case!(case_2sxe, r#".\2SXE"#);
case!(case_2xxw, r#".\2XXW"#);
case!(case_33x3, r#".\33X3"#);
case!(case_35kp, r#".\35KP"#);
case!(case_36f6, r#".\36F6"#);
case!(case_3alj, r#".\3ALJ"#);
case!(case_3gzx, r#".\3GZX"#);
case!(case_3hfz, r#".\3HFZ"#, fail: true);
case!(case_3myt, r#".\3MYT"#);
case!(case_3r3p, r#".\3R3P"#);
case!(case_3rln_00, r#".\3RLN\00"#);
case!(case_3rln_01, r#".\3RLN\01"#);
case!(case_3rln_02, r#".\3RLN\02"#);
case!(case_3rln_03, r#".\3RLN\03"#);
case!(case_3rln_04, r#".\3RLN\04"#);
case!(case_3rln_05, r#".\3RLN\05"#);
case!(case_3uys, r#".\3UYS"#);
case!(case_4abk, r#".\4ABK"#);
case!(case_4cqq, r#".\4CQQ"#);
case!(case_4ejs, r#".\4EJS"#, fail: true);
case!(case_4fj6, r#".\4FJ6"#);
case!(case_4gc6, r#".\4GC6"#, skip: true);
case!(case_4h7k, r#".\4H7K"#, fail: true);
case!(case_4hvu, r#".\4HVU"#, fail: true);
case!(case_4jvg, r#".\4JVG"#, fail: true);
case!(case_4muz_00, r#".\4MUZ\00"#);
case!(case_4muz_01, r#".\4MUZ\01"#);
case!(case_4muz_02, r#".\4MUZ\02"#);
case!(case_4q9f, r#".\4Q9F"#);
case!(case_4qfq, r#".\4QFQ"#);
case!(case_4rwc, r#".\4RWC"#);
case!(case_4uyu, r#".\4UYU"#);
case!(case_4v8u, r#".\4V8U"#);
case!(case_4wa9, r#".\4WA9"#);
case!(case_4zym, r#".\4ZYM"#);
case!(case_52dl, r#".\52DL"#);
case!(case_54t7, r#".\54T7"#);
case!(case_55wf, r#".\55WF"#, fail: true);
case!(case_565n, r#".\565N"#);
case!(case_57h4, r#".\57H4"#);
case!(case_58mp, r#".\58MP"#);
case!(case_5bvj, r#".\5BVJ"#);
case!(case_5c5m, r#".\5C5M"#);
case!(case_5gbf, r#".\5GBF"#);
case!(case_5kje, r#".\5KJE"#, skip: true);
case!(case_5llu, r#".\5LLU"#, skip: true, fail: true);
case!(case_5mud, r#".\5MUD"#);
case!(case_5nyz, r#".\5NYZ"#);
case!(case_5t43, r#".\5T43"#);
case!(case_5trb, r#".\5TRB"#, fail: true);
case!(case_5tym, r#".\5TYM"#, skip: true);
case!(case_5u3a, r#".\5U3A"#, fail: true);
case!(case_5we3, r#".\5WE3"#);
case!(case_62ez, r#".\62EZ"#, fail: true);
case!(case_652z, r#".\652Z"#);
case!(case_65wh, r#".\65WH"#);
case!(case_6bct, r#".\6BCT"#);
case!(case_6bfj, r#".\6BFJ"#);
case!(case_6ca3, r#".\6CA3"#);
case!(case_6ck3, r#".\6CK3"#);
case!(case_6fwr, r#".\6FWR"#);
case!(case_6h3v, r#".\6H3V"#);
case!(case_6hb6, r#".\6HB6"#);
case!(case_6jqw, r#".\6JQW"#);
case!(case_6jtt, r#".\6JTT"#, fail: true);
case!(case_6jwb, r#".\6JWB"#);
case!(case_6kgn, r#".\6KGN"#);
case!(case_6lvf, r#".\6LVF"#);
case!(case_6m2f, r#".\6M2F"#);
case!(case_6pbe, r#".\6PBE"#);
case!(case_6s55, r#".\6S55"#, fail: true);
case!(case_6sla, r#".\6SLA"#);
case!(case_6vjk, r#".\6VJK"#);
case!(case_6wlz, r#".\6WLZ"#);
case!(case_6wpf, r#".\6WPF"#);
case!(case_6xdy, r#".\6XDY"#);
case!(case_6zkb, r#".\6ZKB"#);
case!(case_735y, r#".\735Y"#);
case!(case_74h7, r#".\74H7"#);
case!(case_753e, r#".\753E"#);
case!(case_7a4e, r#".\7A4E"#);
case!(case_7bmt, r#".\7BMT"#);
case!(case_7bub, r#".\7BUB"#);
case!(case_7fwl, r#".\7FWL"#);
case!(case_7lbh, r#".\7LBH"#, fail: true);
case!(case_7mnf, r#".\7MNF"#, fail: true);
case!(case_7t8x, r#".\7T8X"#);
case!(case_7tmg, r#".\7TMG"#);
case!(case_7w2p, r#".\7W2P"#);
case!(case_7z25, r#".\7Z25"#);
case!(case_7zz5, r#".\7ZZ5"#);
case!(case_82an, r#".\82AN"#);
case!(case_87e4, r#".\87E4"#, skip: true);
case!(case_8cwc, r#".\8CWC"#);
case!(case_8g76, r#".\8G76"#);
case!(case_8kb6, r#".\8KB6"#);
case!(case_8mk2, r#".\8MK2"#);
case!(case_8qbe, r#".\8QBE"#);
case!(case_8udb, r#".\8UDB"#, skip: true);
case!(case_8xdj, r#".\8XDJ"#, fail: true);
case!(case_8xyn, r#".\8XYN"#);
case!(case_93jh, r#".\93JH"#);
case!(case_93wf, r#".\93WF"#);
case!(case_96l6, r#".\96L6"#);
case!(case_96nn_00, r#".\96NN\00"#);
case!(case_96nn_01, r#".\96NN\01"#);
case!(case_98yd, r#".\98YD"#);
case!(case_9bxh, r#".\9BXH"#);
case!(case_9c9n, r#".\9C9N"#, skip: true, fail: true);
case!(case_9cwy, r#".\9CWY"#, fail: true);
case!(case_9dxl, r#".\9DXL"#);
case!(case_9fmg, r#".\9FMG"#);
case!(case_9hcy, r#".\9HCY"#, fail: true);
case!(case_9j7a, r#".\9J7A"#);
case!(case_9jba, r#".\9JBA"#, skip: true, fail: true);
case!(case_9kax, r#".\9KAX"#);
case!(case_9kbc, r#".\9KBC"#, fail: true);
case!(case_9mag, r#".\9MAG"#, fail: true);
case!(case_9mma, r#".\9MMA"#, fail: true);
case!(case_9mmw, r#".\9MMW"#);
case!(case_9mqt_00, r#".\9MQT\00"#);
case!(case_9mqt_01, r#".\9MQT\01"#, fail: true);
case!(case_9sa2, r#".\9SA2"#);
case!(case_9shh, r#".\9SHH"#, skip: true);
case!(case_9tfx, r#".\9TFX"#);
case!(case_9u5k, r#".\9U5K"#);
case!(case_9wxw, r#".\9WXW"#);
case!(case_9yrd, r#".\9YRD"#);
case!(case_a2m4, r#".\A2M4"#);
case!(case_a6f9, r#".\A6F9"#);
case!(case_a984, r#".\A984"#);
case!(case_ab8u, r#".\AB8U"#);
case!(case_avm7, r#".\AVM7"#);
case!(case_az63, r#".\AZ63"#);
case!(case_azw3, r#".\AZW3"#);
case!(case_b3hg, r#".\B3HG"#);
case!(case_b63p, r#".\B63P"#, fail: true);
case!(case_bd7l, r#".\BD7L"#, fail: true);
case!(case_bec7, r#".\BEC7"#);
case!(case_bf9h, r#".\BF9H"#, fail: true);
case!(case_bs4k, r#".\BS4K"#, fail: true);
case!(case_bu8l, r#".\BU8L"#);
case!(case_c2dt, r#".\C2DT"#);
case!(case_c2sp, r#".\C2SP"#, fail: true);
case!(case_c4hz, r#".\C4HZ"#);
case!(case_cc74, r#".\CC74"#);
case!(case_cfd4, r#".\CFD4"#);
case!(case_cml9, r#".\CML9"#, fail: true);
case!(case_cn3r, r#".\CN3R"#);
case!(case_cpz3, r#".\CPZ3"#);
case!(case_cq3w, r#".\CQ3W"#, fail: true);
case!(case_ct4q, r#".\CT4Q"#);
case!(case_ctn5, r#".\CTN5"#, skip: true, fail: true);
case!(case_cup7, r#".\CUP7"#);
case!(case_cvw2, r#".\CVW2"#, skip: true, fail: true);
case!(case_cxx2, r#".\CXX2"#, fail: true);
case!(case_d49q, r#".\D49Q"#, fail: true);
case!(case_d83l, r#".\D83L"#);
case!(case_d88j, r#".\D88J"#);
case!(case_d9tu, r#".\D9TU"#);
case!(case_dbg4, r#".\DBG4"#, skip: true);
case!(case_dc7x, r#".\DC7X"#);
case!(case_de56_00, r#".\DE56\00"#);
case!(case_de56_01, r#".\DE56\01"#);
case!(case_de56_02, r#".\DE56\02"#);
case!(case_de56_03, r#".\DE56\03"#);
case!(case_de56_04, r#".\DE56\04"#);
case!(case_de56_05, r#".\DE56\05"#);
case!(case_dff7, r#".\DFF7"#);
case!(case_dhp8, r#".\DHP8"#);
case!(case_dk3j, r#".\DK3J"#);
case!(case_dk4h, r#".\DK4H"#, fail: true);
case!(case_dk95_00, r#".\DK95\00"#);
case!(case_dk95_01, r#".\DK95\01"#, fail: true);
case!(case_dk95_02, r#".\DK95\02"#);
case!(case_dk95_03, r#".\DK95\03"#);
case!(case_dk95_04, r#".\DK95\04"#);
case!(case_dk95_05, r#".\DK95\05"#);
case!(case_dk95_06, r#".\DK95\06"#, fail: true);
case!(case_dk95_07, r#".\DK95\07"#);
case!(case_dk95_08, r#".\DK95\08"#);
case!(case_dmg6, r#".\DMG6"#, fail: true);
case!(case_dwx9, r#".\DWX9"#);
case!(case_e76z, r#".\E76Z"#);
case!(case_eb22, r#".\EB22"#, fail: true);
case!(case_ehf6, r#".\EHF6"#);
case!(case_ew3v, r#".\EW3V"#, fail: true);
case!(case_ex5h, r#".\EX5H"#);
case!(case_exg3, r#".\EXG3"#);
case!(case_f2c7, r#".\F2C7"#);
case!(case_f3cp, r#".\F3CP"#);
case!(case_f6mc, r#".\F6MC"#);
case!(case_f8f9, r#".\F8F9"#);
case!(case_fbc9, r#".\FBC9"#);
case!(case_fh7j, r#".\FH7J"#);
case!(case_fp8r, r#".\FP8R"#);
case!(case_fq7f, r#".\FQ7F"#);
case!(case_frk4, r#".\FRK4"#);
case!(case_fta2, r#".\FTA2"#);
case!(case_fup4, r#".\FUP4"#);
case!(case_g4rs, r#".\G4RS"#, skip: true);
case!(case_g5u8, r#".\G5U8"#, fail: true);
case!(case_g7je, r#".\G7JE"#, fail: true);
case!(case_g992, r#".\G992"#);
case!(case_g9hc, r#".\G9HC"#, fail: true);
case!(case_gdy7, r#".\GDY7"#, fail: true);
case!(case_gh63, r#".\GH63"#);
case!(case_gt5m, r#".\GT5M"#, fail: true);
case!(case_h2rw, r#".\H2RW"#);
case!(case_h3z8, r#".\H3Z8"#);
case!(case_h7j7, r#".\H7J7"#, fail: true);
case!(case_h7tq, r#".\H7TQ"#, fail: true);
case!(case_hm87_00, r#".\HM87\00"#);
case!(case_hm87_01, r#".\HM87\01"#);
case!(case_hmk4, r#".\HMK4"#);
case!(case_hmq5, r#".\HMQ5"#);
case!(case_hre5, r#".\HRE5"#, skip: true, fail: true);
case!(case_hs5t, r#".\HS5T"#);
case!(case_hu3p, r#".\HU3P"#, fail: true);
case!(case_hwv9, r#".\HWV9"#);
case!(case_j3bt, r#".\J3BT"#, skip: true);
case!(case_j5uc, r#".\J5UC"#);
case!(case_j7pz, r#".\J7PZ"#);
case!(case_j7vc, r#".\J7VC"#);
case!(case_j9hz, r#".\J9HZ"#);
case!(case_jef9_00, r#".\JEF9\00"#);
case!(case_jef9_01, r#".\JEF9\01"#);
case!(case_jef9_02, r#".\JEF9\02"#);
case!(case_jhb9, r#".\JHB9"#);
case!(case_jkf3, r#".\JKF3"#, fail: true);
case!(case_jq4r, r#".\JQ4R"#);
case!(case_jr7v, r#".\JR7V"#);
case!(case_js2j, r#".\JS2J"#);
case!(case_jtv5, r#".\JTV5"#);
case!(case_jy7z, r#".\JY7Z"#, fail: true);
case!(case_k3wx, r#".\K3WX"#);
case!(case_k4su, r#".\K4SU"#);
case!(case_k527, r#".\K527"#);
case!(case_k54u, r#".\K54U"#);
case!(case_k858, r#".\K858"#);
case!(case_kh5v_00, r#".\KH5V\00"#);
case!(case_kh5v_01, r#".\KH5V\01"#);
case!(case_kh5v_02, r#".\KH5V\02"#);
case!(case_kk5p, r#".\KK5P"#);
case!(case_kmk3, r#".\KMK3"#);
case!(case_ks4u, r#".\KS4U"#, fail: true);
case!(case_kss4, r#".\KSS4"#);
case!(case_l24t_00, r#".\L24T\00"#);
case!(case_l24t_01, r#".\L24T\01"#);
case!(case_l383, r#".\L383"#);
case!(case_l94m, r#".\L94M"#);
case!(case_l9u5, r#".\L9U5"#, skip: true);
case!(case_le5a, r#".\LE5A"#, skip: true);
case!(case_lhl4, r#".\LHL4"#, fail: true);
case!(case_lp6e, r#".\LP6E"#, skip: true);
case!(case_lqz7, r#".\LQZ7"#, skip: true);
case!(case_lx3p, r#".\LX3P"#);
case!(case_m29m, r#".\M29M"#);
case!(case_m2n8_00, r#".\M2N8\00"#);
case!(case_m2n8_01, r#".\M2N8\01"#);
case!(case_m5c3, r#".\M5C3"#);
case!(case_m5dy, r#".\M5DY"#, skip: true);
case!(case_m6yh, r#".\M6YH"#);
case!(case_m7a3, r#".\M7A3"#);
case!(case_m7nx, r#".\M7NX"#);
case!(case_m9b4, r#".\M9B4"#);
case!(case_mjs9, r#".\MJS9"#);
case!(case_mus6_00, r#".\MUS6\00"#, fail: true);
case!(case_mus6_01, r#".\MUS6\01"#, fail: true);
case!(case_mus6_02, r#".\MUS6\02"#);
case!(case_mus6_03, r#".\MUS6\03"#);
case!(case_mus6_04, r#".\MUS6\04"#);
case!(case_mus6_05, r#".\MUS6\05"#);
case!(case_mus6_06, r#".\MUS6\06"#);
case!(case_mxs3, r#".\MXS3"#);
case!(case_myw6, r#".\MYW6"#);
case!(case_mzx3, r#".\MZX3"#, skip: true);
case!(case_n4jp, r#".\N4JP"#, fail: true);
case!(case_n782, r#".\N782"#, fail: true);
case!(case_nat4, r#".\NAT4"#, skip: true);
case!(case_nb6z, r#".\NB6Z"#);
case!(case_nhx8, r#".\NHX8"#);
case!(case_nj66, r#".\NJ66"#);
case!(case_nkf9, r#".\NKF9"#);
case!(case_np9h, r#".\NP9H"#);
case!(case_p2ad, r#".\P2AD"#);
case!(case_p2eq, r#".\P2EQ"#, fail: true);
case!(case_p76l, r#".\P76L"#);
case!(case_p94k, r#".\P94K"#);
case!(case_pbj2, r#".\PBJ2"#);
case!(case_prh3, r#".\PRH3"#, skip: true);
case!(case_puw8, r#".\PUW8"#);
case!(case_pw8x, r#".\PW8X"#);
case!(case_q4cl, r#".\Q4CL"#, fail: true);
case!(case_q5mg, r#".\Q5MG"#);
case!(case_q88a, r#".\Q88A"#, skip: true);
case!(case_q8ad, r#".\Q8AD"#);
case!(case_q9wf, r#".\Q9WF"#);
case!(case_qb6e, r#".\QB6E"#, fail: true);
case!(case_qf4y, r#".\QF4Y"#);
case!(case_qlj7, r#".\QLJ7"#, skip: true, fail: true);
case!(case_qt73, r#".\QT73"#);
case!(case_r4yg, r#".\R4YG"#);
case!(case_r52l, r#".\R52L"#);
case!(case_rhx7, r#".\RHX7"#, fail: true);
case!(case_rlu9, r#".\RLU9"#);
case!(case_rr7f, r#".\RR7F"#);
case!(case_rtp8, r#".\RTP8"#);
case!(case_rxy3, r#".\RXY3"#, skip: true, fail: true);
case!(case_rzp5, r#".\RZP5"#);
case!(case_rzt7, r#".\RZT7"#);
case!(case_s3pd, r#".\S3PD"#);
case!(case_s4gj, r#".\S4GJ"#, fail: true);
case!(case_s4jq, r#".\S4JQ"#);
case!(case_s4t7, r#".\S4T7"#);
case!(case_s7bg, r#".\S7BG"#);
case!(case_s98z, r#".\S98Z"#, skip: true, fail: true);
case!(case_s9e8, r#".\S9E8"#);
case!(case_sbg9, r#".\SBG9"#);
case!(case_sf5v, r#".\SF5V"#, skip: true, fail: true);
case!(case_ske5, r#".\SKE5"#);
case!(case_sm9w_00, r#".\SM9W\00"#);
case!(case_sm9w_01, r#".\SM9W\01"#);
case!(case_sr86, r#".\SR86"#, fail: true);
case!(case_ssw6, r#".\SSW6"#, skip: true);
case!(case_su5z, r#".\SU5Z"#, fail: true);
case!(case_su74, r#".\SU74"#, fail: true);
case!(case_sy6v, r#".\SY6V"#, fail: true);
case!(case_syw4, r#".\SYW4"#);
case!(case_t26h, r#".\T26H"#);
case!(case_t4yy, r#".\T4YY"#, skip: true);
case!(case_t5n4, r#".\T5N4"#);
case!(case_t833, r#".\T833"#, fail: true);
case!(case_td5n, r#".\TD5N"#, fail: true);
case!(case_te2a, r#".\TE2A"#);
case!(case_tl85, r#".\TL85"#);
case!(case_ts54, r#".\TS54"#);
case!(case_u3c3, r#".\U3C3"#);
case!(case_u3xv, r#".\U3XV"#);
case!(case_u44r, r#".\U44R"#, fail: true);
case!(case_u99r, r#".\U99R"#, fail: true);
case!(case_u9ns, r#".\U9NS"#);
case!(case_udm2, r#".\UDM2"#);
case!(case_udr7, r#".\UDR7"#, skip: true);
case!(case_ugm3, r#".\UGM3"#);
case!(case_ukk6_00, r#".\UKK6\00"#);
case!(case_ukk6_01, r#".\UKK6\01"#);
case!(case_ukk6_02, r#".\UKK6\02"#);
case!(case_ut92, r#".\UT92"#);
case!(case_uv7q, r#".\UV7Q"#);
case!(case_v55r, r#".\V55R"#);
case!(case_v9d5, r#".\V9D5"#);
case!(case_vjp3_00, r#".\VJP3\00"#, fail: true);
case!(case_vjp3_01, r#".\VJP3\01"#);
case!(case_w42u, r#".\W42U"#);
case!(case_w4tn, r#".\W4TN"#);
case!(case_w5vh, r#".\W5VH"#);
case!(case_w9l4, r#".\W9L4"#, skip: true, fail: true);
case!(case_wz62, r#".\WZ62"#);
case!(case_x38w, r#".\X38W"#);
case!(case_x4qw, r#".\X4QW"#, fail: true);
case!(case_x8dw, r#".\X8DW"#);
case!(case_xlq9, r#".\XLQ9"#);
case!(case_xv9v, r#".\XV9V"#);
case!(case_xw4d, r#".\XW4D"#);
case!(case_y2gn, r#".\Y2GN"#);
case!(case_y79y_000, r#".\Y79Y\000"#, fail: true);
case!(case_y79y_001, r#".\Y79Y\001"#);
case!(case_y79y_002, r#".\Y79Y\002"#);
case!(case_y79y_003, r#".\Y79Y\003"#, fail: true);
case!(case_y79y_004, r#".\Y79Y\004"#, fail: true);
case!(case_y79y_005, r#".\Y79Y\005"#, fail: true);
case!(case_y79y_006, r#".\Y79Y\006"#, fail: true);
case!(case_y79y_007, r#".\Y79Y\007"#, fail: true);
case!(case_y79y_008, r#".\Y79Y\008"#, fail: true);
case!(case_y79y_009, r#".\Y79Y\009"#, fail: true);
case!(case_y79y_010, r#".\Y79Y\010"#);
case!(case_yd5x, r#".\YD5X"#);
case!(case_yjv2, r#".\YJV2"#, fail: true);
case!(case_z67p, r#".\Z67P"#);
case!(case_z9m4, r#".\Z9M4"#);
case!(case_zcz6, r#".\ZCZ6"#, fail: true);
case!(case_zf4x, r#".\ZF4X"#);
case!(case_zh7c, r#".\ZH7C"#);
case!(case_zk9h, r#".\ZK9H"#);
case!(case_zl4z, r#".\ZL4Z"#, skip: true, fail: true);
case!(case_zvh3, r#".\ZVH3"#, fail: true);
case!(case_zwk4, r#".\ZWK4"#);
case!(case_zxt5, r#".\ZXT5"#, fail: true);
