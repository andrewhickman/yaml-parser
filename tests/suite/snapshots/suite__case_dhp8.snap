---
source: tests/suite/cases.gen.rs
expression: tokens
---
- token: Indent
  start:
    index: 0
    line: 0
    column: 0
  end:
    index: 0
    line: 0
    column: 0
- token: SequenceStart
  start:
    index: 0
    line: 0
    column: 0
  end:
    index: 1
    line: 0
    column: 1
- token: Scalar
  start:
    index: 1
    line: 0
    column: 1
  end:
    index: 4
    line: 0
    column: 4
- token: CollectionEntry
  start:
    index: 4
    line: 0
    column: 4
  end:
    index: 5
    line: 0
    column: 5
- token: Separator
  start:
    index: 5
    line: 0
    column: 5
  end:
    index: 6
    line: 0
    column: 6
- token: Scalar
  start:
    index: 6
    line: 0
    column: 6
  end:
    index: 9
    line: 0
    column: 9
- token: CollectionEntry
  start:
    index: 9
    line: 0
    column: 9
  end:
    index: 10
    line: 0
    column: 10
- token: Separator
  start:
    index: 10
    line: 0
    column: 10
  end:
    index: 11
    line: 0
    column: 11
- token: Scalar
  start:
    index: 11
    line: 0
    column: 11
  end:
    index: 13
    line: 0
    column: 13
- token: SequenceEnd
  start:
    index: 13
    line: 0
    column: 13
  end:
    index: 14
    line: 0
    column: 14
- token: Break
  start:
    index: 14
    line: 0
    column: 14
  end:
    index: 16
    line: 1
    column: 0

